### PR TITLE
feat(doc): add product pages for NW Core, NW Edge and M&Q

### PR DESCRIPTION
**Pull request by RoRoJ**
Add product pages for network core and edge products, and mnq

### CODE REVIEW

**Review by ldecarvalho-doc — APPROVED** [review]

File: network/public-gateways/index.mdx [hunk 1]
```diff
@@ -1,8 +1,84 @@
...
+    <SummaryCard
+        title="How-tos"
+        icon="help-circle-outline"
+        description="Check our guides to creating and configuring Public Gateways."
+        label="View How-tos"
+        url="/network/public-gateways/how-to/"
+    />
+</Grid>
+
+
```
Comment (bene2k1) [inline network/public-gateways/index.mdx:49]:
```suggestion
```

File: network/load-balancer/index.mdx [hunk 2]
```diff
@@ -1,8 +1,88 @@
...
+        label="Read more"
+    />
+    <DefaultCard
+        title="Setting up Traefik v2 and cert-manager on Kapsule"
+        url="/tutorials/traefik-v2-cert-manager/"
+        label="Read more"
+    />
+</Grid>
+
+
```
Comment (bene2k1) [inline network/load-balancer/index.mdx:74]:
```suggestion
```

File: network/load-balancer/index.mdx [hunk 3]
```diff
@@ -1,8 +1,88 @@
...
+        <SummaryCard
+        title="Additional Resources"
+        icon="book-open-outline"
+        description="Go beyond the basics with detailed, in-depth guides to configuring your Load Balancer."
+        label="Read more"
+        url="/network/load-balancer/reference-content/"
+    />
+</Grid>
+
+
```
Comment (bene2k1) [inline network/load-balancer/index.mdx:48]:
```suggestion
```

File: network/load-balancer/index.mdx (hunk 2 above)
Comment (bene2k1) [reply network/load-balancer/index.mdx:74]: remove other double new lines maybe?

**Review by bene2k1 — APPROVED** [review]

File: network/load-balancer/index.mdx [hunk 4]
```diff
@@ -1,8 +1,86 @@
...
+    />
+    <SummaryCard
+        title="How-tos"
+        icon="help-circle-outline"
+        description="Check our guides to creating and configuring Load Balancers."
+        label="View How-tos"
+        url="/network/load-balancer/how-to/"
+    />
+        <SummaryCard
+        title="Additional Resources"
```
Comment (jcirinosclwy) [inline network/load-balancer/index.mdx:40]: Additional Content

File: network/load-balancer/index.mdx [hunk 5]
```diff
@@ -1,8 +1,86 @@
...
+        icon="book-open-outline"
+        description="Go beyond the basics with detailed, in-depth guides to configuring your Load Balancer."
+        label="Read more"
```
Comment (jcirinosclwy) [inline network/load-balancer/index.mdx:43]: View Additional Content

File: network/vpc/index.mdx [hunk 6]
```diff
@@ -1,8 +1,90 @@
...
+    />
+    <SummaryCard
+        title="How-tos"
+        icon="help-circle-outline"
+        description="Check our guides to creating and configuring VPCs and Private Networks."
+        label="View How-tos"
+        url="/network/vpc/how-to/"
+    />
+    <SummaryCard
+        title="Additional Resources"
```
Comment (jcirinosclwy) [inline network/vpc/index.mdx:47]: Additional Content

File: network/vpc/index.mdx [hunk 7]
```diff
@@ -1,8 +1,90 @@
...
+        icon="book-open-outline"
+        description="Guide to the migration from zoned Private Networks to regionalized VPCs."
+        label="View Additional Resources"
```
Comment (jcirinosclwy) [inline network/vpc/index.mdx:50]: View Additional Content

File: serverless/messaging/index.mdx [hunk 8]
```diff
@@ -1,8 +1,83 @@
...
+    />
+    <SummaryCard
+        title="How-tos"
+        icon="help-circle-outline"
+        description="Check our guides to activate and configure Messaging and Queuing for SQS, SNS or NATS."
+        label="View How-tos"
+        url="/serverless/messaging/how-to/"
+    />
+        <SummaryCard
+        title="Additional Resources"
```
Comment (jcirinosclwy) [inline serverless/messaging/index.mdx:40]: Additional Content

File: serverless/messaging/index.mdx [hunk 9]
```diff
@@ -1,8 +1,83 @@
...
+        icon="book-open-outline"
+        description="Take a deeper dive into using Messaging and Queuing with SQS, SNS or NATS."
+        label="View Additional Resources"
```
Comment (jcirinosclwy) [inline serverless/messaging/index.mdx:43]: View Additional Content

**Pull request merged**